### PR TITLE
Remove custom SIGINT handler from offline node

### PR DESCRIPTION
**Pull request by ojura**
I have singled here out the first commit from the other pull request.

### CODE REVIEW

**Comment by wohe** [comment]
Copying the rationale from #468: There is the obvious problem before #230 that trajectories are not correctly finished which does cause crashes. This is fixed by the change of `return` to `break` which you retain. It does not need a signal handler.

File: cartographer_ros/cartographer_ros/offline_node_main.cc [hunk 1]
```diff
@@ -252,9 +248,7 @@ int main(int argc, char** argv) {
       << "-configuration_basename is missing.";
   CHECK(!FLAGS_bag_filenames.empty()) << "-bag_filenames is missing.";
 
-  std::signal(SIGINT, &::cartographer_ros::SigintHandler);
-  ::ros::init(argc, argv, "cartographer_offline_node",
-              ::ros::init_options::NoSigintHandler);
+  ::ros::init(argc, argv, "cartographer_offline_node");
```
Comment (wohe) [inline cartographer_ros/cartographer_ros/offline_node_main.cc]: I think the `#include <csignal>` can also be removed.

Comment (ojura) [inline cartographer_ros/cartographer_ros/offline_node_main.cc]: Updated

**Comment by ojura** [comment]
> trajectories are not correctly finished which does cause crashes

Sounds like `Node` should check in its destructor if there are any unfinalized trajectories.

**Comment by wohe** [comment]
That's what I meant, it is `CHECK`ed. In the `OrderedMultiQueue` though: https://github.com/googlecartographer/cartographer/blob/97bb74157f14267c43f6b0c44eda19e7ed6336e1/cartographer/sensor/ordered_multi_queue.cc#L45

**Pull request merged**